packages/e2e-tests/specs/embedding.test.js

```diff
@@ -124,6 +125,12 @@ const MOCK_RESPONSES = [
 		match: createEmbeddingMatcher( 'https://twitter.com/wooyaygutenberg123454312' ),
 		onRequestMatch: createJSONResponse( MOCK_CANT_EMBED_RESPONSE ),
 	},
+	// Respond to the instagram URL with a non-image response, doesn't matter what it is,
+	// just make sure the image errors.
+	{
+		match: createURLMatcher( 'https://www.instagram.com/p/Bvl97o2AK6x/' ),
```

Comment:
Should we remove the other matcher?

https://github.com/WordPress/gutenberg/blob/82a5ecc2c0147df2c74e0ad09fde9753049cc968/packages/e2e-tests/specs/embedding.test.js#L105-L106

Reply:
it doesn't seem to be used at the moment

Reply:
No, one is to match the embed preview request for the URL and one is to match the URL itself. The image block creates a new image, and sets the URL as the source. If the image errors, it replaces itself with an appropriate embed block if there is one. So we need to mock the instagram URL, and the embed proxy call that generates the preview.

Reply:
Awesome, it makes sense now :)